types/imagemin/index.d.ts

```diff
@@ -1,23 +1,26 @@
-// Type definitions for imagemin 6.0
+// Type definitions for imagemin 7.0
 // Project: https://github.com/imagemin/imagemin#readme
 // Definitions by: Romain Faust <https://github.com/romain-faust>
+//                 Jeff Chan <https://github.com/hkjeffchan>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /// <reference types="node" />
 
-declare function imagemin(input: ReadonlyArray<string>, outputOrOptions?: string | imagemin.Options): Promise<imagemin.Result[]>;
-declare function imagemin(input: ReadonlyArray<string>, output?: string, options?: imagemin.Options): Promise<imagemin.Result[]>;
+declare function imagemin(input: string[], options?: imagemin.Options): Promise<imagemin.Result[]>;
```

Comment:
Any reason you've replaced `ReadonlyArray<string>` with `string[]` ?

Reply:
No, you may keep it ReadonlyArray<string>